libs/Bindings.c

```diff
@@ -827,8 +827,7 @@ KeySym FvwmStringToKeysym(Display *dpy, char *key)
 	}
 	else
 	{
-		s = alloca(strlen(key) + 1);
-		strcpy(s, key);
+		snprintf(s, sizeof(key), key);
```

Comment:
This bit reads wrong: it writes to an un-initialized pointer `s`.  I believe it should be converted to either strdup or a local buffer (i.e. declaring `s` as `char s[32]` or any other sensible maximum size.)

Reply:
Oops.  You're entirely correct.  Will fix.